schemas/trademark-registration-schema.json

```diff
@@ -0,0 +1,200 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A Trademark is a recognizable sign, design, or expression which identifies products or services of a particular source from those of others. This schema is actually for the trademark registration, i.e. not just the mark details, but the record of when it was registered and who holds the trademark. See also http://www.wipo.int/standards/XMLSchema/HTML/ST96TechnicalSpecification/V0-11/TrademarkComponents.html",
+  "properties": {
+    "register": {
+      "description": "The register on which the trademark is listed",
+      "type": "string"
+    },
+    "holders": {
+      "type": "array",
+      "description": "Details of the holder(s) of the trademark (sometimes called the owners)",
+      "items": {
+        "$ref": "includes/entity.json"
+      }
+    },
+    "correspondent": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the correspondent (i.e. contact person)"
+    },
+    "representative": {
+      "$ref": "includes/entity.json",
+      "description": "Details of the representative"
+    },
+    "mark_details": {
+      "description": "Details of the trademark itself (rather than the registration)",
+      "type": "object",
+      "properties": {
+        "mark_text": {
+          "description": "Text of the trademark",
+          "type": "string"
+        },
+        "mark_form": {
+          "description": "The form(s) (i.e. category) of the mark. Sensory marks include smells and sounds",
+          "type": "string",
+          "enum": [
+            "text",
+            "image",
+            "sensory_mark",
+            "three_d_mark"
+          ]
+        },
+        "mark_type": {
+          "description": "Free text description of mark type (as provided by register), e.g. 'Illustration', 'Typeset: Word(s)/letter(s)/number(s)'",
```

Comment:
Do we need to say "(as provided by register)" here? There are other places where it's "as provided by register", e.g. `status` below.